Parsers/InfobloxNIOS/InfobloxNIOS.txt

```diff
@@ -191,7 +184,7 @@ let dnszone = RawData
         Message = tostring(dnszone_substring[1])
     | project-away RawData_subString, dnszone_substring, dnszone;
 let dnsclient = RawData 
```

Comment:
PLease check regex and field parsing

![image](https://user-images.githubusercontent.com/11397335/88125689-ac983800-cb84-11ea-81c5-9ad9550be086.png)

Reply:
This should be resolved with the updated data sample